### PR TITLE
Mapserver working with python3

### PR DESCRIPTION
**Pull request by cphyc**
This PR brings back the mapserver command line utils for Python3.

## PR Summary

This basically requires bottle as an extra dependencies. If you have a more clever idea, I'd be open to hear it. Also I noticed I had trouble running bottle with the `rocket` server. In the doc, they say you can use `auto` instead.

## PR Checklist

- [x] Code passes flake8 checker

### CODE REVIEW

**Comment by ngoldbaum** [comment]
@bwkeller does no longer vendoring bottle and leaflet have any annoying impacts on the debian packages you're maintaining?

**Comment by bwkeller** [comment]
Actually, that would make my life somewhat easier.  Debian prefers to have packages that rely on JS libraries to use the debian version rather than vendor their own versions, so I've already been using a patch that already strips out the minified leaflet.

File: yt/utilities/command_line.py [hunk 1]
```diff
@@ -754,21 +751,18 @@ def __call__(self, args):
             p = SlicePlot(ds, args.axis, args.field)
         from yt.visualization.mapserver.pannable_map import PannableMapServer
         PannableMapServer(p.data_source, args.field)
-        import yt.extern.bottle as bottle
+        import bottle
```
Comment (ngoldbaum) [inline yt/utilities/command_line.py]: Can you wrap this in a `try, except ImortError block` suggesting to install bottle of importing it doesn’t work?

Comment (cphyc) [reply yt/utilities/command_line.py]: Something like:
```
        try:
            import bottle
        except ImportError:
            raise RuntimeError(
                "The mapserver requires the package `bottle' to be "
                "installed. Please install using `pip install bottle`."
            )
```

Comment (ngoldbaum) [reply yt/utilities/command_line.py]: yup

**Pull request merged**